packages/launch/src/launch/index.tsx

```diff
@@ -1,45 +1,48 @@
 /**
  * External dependencies
  */
-import React, { useCallback } from 'react';
+import React from 'react';
```

Comment:
Should it be changed to `import * as React from 'react'` ?

Reply:
Updated for all the package in https://github.com/Automattic/wp-calypso/pull/47601/commits/2d9af7a3079d2968ccc129e0faa74a96d0e492c8